visio_ml/src/modules/compile_results.py

```diff
@@ -0,0 +1,49 @@
+import cv2
+import firebase_admin
+import datetime
+
+class ResultCompiler:
+    PATH_TO_CREDENTIALS = "" # TODO: Add file path to firebase credentials
+
+    # results_db_path: URL to the realtime database for results
+    # results_destination_path: destination file in firebase storage for results
+    # video_db_name: name of the realtime database for videos
+    def __init__(self, exercise, body_part, results_db_path, results_destination_path, video_db_name, frame_rate):
+        self.cred = firebase_admin.credentials.Certificate(self.PATH_TO_CREDENTIALS)
+        self.exercise = exercise
+        self.body_part = body_part
+        self.results_db_path = results_db_path
+        self.results_destination_path = results_destination_path
+        self.video_db_name = video_db_name
+        self.results = []
+        self.total = 0
```

Comment:
Just had a thought, for shoulder, user can do the same exercise on both side at the same time. So it's better to keep results and total as a dictionary.
self.result = {'left': [ ], 'right': [ ]}.
For hip, user can only do one side at a time:
self.result = {'left': [ ]}